dokuwiki-text/src/test/resources/dokuwikitext/dokuwiki-contents/conf/dokuwiki.php

```diff
@@ -1,177 +0,0 @@
-<?php
```

Comment:
I think you should actually use that file, for example I see that data/ folder is actually configurable and indicated in this file. Same comment fo the lang property which should probably be indicated in the documents.

Also it might be interresting to allow changing some of the XWiki configuration based on this file (import mail setup, etc.).

Reply:
Great idea. I'm just curious about the configuration that the filter framework supports, currently.